saga-format/src/main/java/io/servicecomb/saga/format/JsonRestSagaRequest.java

```diff
@@ -38,16 +38,19 @@ public JsonRestSagaRequest(
       @JsonProperty("transaction") JacksonRestTransaction transaction,
       @JsonProperty("compensation") JacksonRestCompensation compensation,
       @JsonProperty("fallback") JacksonFallback fallback,
-      @JsonProperty("parents") String[] parents) {
+      @JsonProperty("parents") String[] parents,
+      @JsonProperty("failRetryDelayMilliseconds") int failRetryDelayMilliseconds) {
 
-    super(id, serviceName, type, transaction, compensation, fallback == null? NOP_TRANSPORT_AWARE_FALLBACK : fallback, parents);
+    super(id, serviceName, type, transaction, compensation,
+        fallback == null ? NOP_TRANSPORT_AWARE_FALLBACK : fallback,
+        parents, failRetryDelayMilliseconds);
```

Comment:
jackson json writes 0 as default value for int. have to check here for 0 and negative (in case of human error) and pass a constant we defined as default here.

Reply:
Done